File: Reference/hello.pl
#!/usr/bin/perl
print "Content-type: text/html\r\n\r\n";
print "<html><head><title>CGI Response</title>
</head>\r\n";
print "<hr>\r\n";
print "<p>\r\n";
print "The data received by the server is:\r\n";
print "<p>\r\n";
print "$ENV{’QUERY_STRING’}\r\n";
print "<p>\r\n";
print "<hr>\r\n";
print "</body></html>\r\n";

File: docker-tools/http/hello.php

<?php
//'REQUEST_URI'
// echo "Content-type: text/html\r\n\r\n";
echo "<html><head><title>CGI Response</title></head>\r\n";
echo "<hr>\r\n";
echo "<p>\r\n";
echo "The data received by the server is:\r\n";
echo "<p>\r\n";
echo $_SERVER["QUERY_STRING"];
// echo " in {’QUERY_STRING’} variable\r\n";
echo "<p>\r\n";
echo "<hr>\r\n";
echo "</body></html>\r\n";
?>
